drizzle-zod/src/mysql/index.ts

```diff
@@ -0,0 +1,243 @@
+import { GetColumnConfig } from 'drizzle-orm';
+import {
+	AnyPgColumn,
+	AnyPgTable,
+	GetTableConfig,
+	PgBigInt53,
+	PgBigInt64,
+	PgBigSerial53,
+	PgBigSerial64,
+	PgBoolean,
+	PgDate,
+	PgDoublePrecision,
+	PgEnumColumn,
+	PgEnumColumnConfig,
+	PgInteger,
+	PgInterval,
+	PgNumeric,
+	PgReal,
+	PgSerial,
+	PgSmallInt,
+	PgSmallSerial,
+	PgText,
+	PgTextConfig,
+	PgTime,
+	PgTimestamp,
+	PgUUID,
+	PgVarchar,
+} from 'drizzle-orm/mysql-core';
+import { getTableColumns } from 'drizzle-orm/pg-core/utils';
```

Comment:
We should use mysql specific helper functions for mysql zod integration

Reply:
Question, would these changes also apply to planet scale? Like could I still import MySQL columns and helpers from MySQL core or would I need to also use the planet scale client?